Fixes issue #1 - inconsistent sidebar styles

melicarls opened the pull request:
- Removed changes to bootstrap CSS
- Found and corrected inconsistent class names in sidebar content (sidebar_title to sidebar-title and sidebar_body to sidebar-body)

melicarls committed:
sidebar font is monospace

melicarls committed:
sidebar lists all have circle bullets

melicarls committed:
undid first attempt

melicarls committed:
fixed inconsistent class names in sidebar

justincastilla commented:
Please describe the changes you made to remove inconsistencies in the class names

melicarls commented:
- Removed changes to bootstrap CSS
- Found and corrected inconsistent class names in sidebar content (sidebar_title to sidebar-title and sidebar_body to sidebar-body in views/archives_sidebar/_content.html.erb)
